src/modules/shortcut_guide/shortcut_guide.cpp

```diff
@@ -391,9 +415,48 @@ void OverlayWindow::init_settings()
         {
             theme.value = std::move(*val);
         }
+        if (auto val = settings.get_string_value(disabledApps.name))
+        {
+            disabledApps.value = std::move(*val);
+            update_disabled_apps();
+        }
     }
     catch (std::exception&)
     {
         // Error while loading from the settings file. Just let default values stay as they are.
     }
 }
+
+bool OverlayWindow::is_disabled_app(wchar_t* exePath)
+{
+    auto exePathUpper = std::wstring(exePath);
+    CharUpperBuffW(exePathUpper.data(), (DWORD)exePathUpper.length());

```

Comment:
This call can potentially crash. This entire function should be in a try catch.

Reply:
How could this crash? This is searching for `exePath` in the array of disabled apps read from settings.

Reply:
It's the previews call that can crash for a NULL input argument.
Main point is: `is_disabled_app` should guarantee to not crash, no matter what is the input